frontend/src/app/main/data/workspace/transforms.cljs

```diff
@@ -787,30 +787,34 @@
 
 (defn update-position
   "Move shapes to a new position"
-  [id position]
-  (dm/assert! (uuid? id))
+  ([id position] (update-position id position nil))
+  ([id position attrs]
```

Comment:
`attrs` should be called `opts` or `options` i think

Reply:
+1 to `options`

Reply:
perfect